src/applications/disability-benefits/all-claims/tests/pages/vaMedicalRecords.unit.spec.jsx

```diff
@@ -188,7 +197,9 @@ describe('VA Medical Records', () => {
     form.unmount();
   });
 
-  it('should submit with all required info', () => {
+  it('should submit with sparse facility info', () => {
+    // Treatment facility is missing a name, date, and city (they are optional). VSRs look at all VAMCs in the
+    // system to assess if it’s relevant to the claim.
```

Comment:
Appreciate the clarifying comment. This is pretty neat, and I'm curious how this check works. Definitely makes things so much more convenient for veterans